deb/openmediavault/usr/share/php/openmediavault/system/system.inc

```diff
@@ -66,11 +98,13 @@ class System {
 			return TRUE;
 		$rootDeviceFile = self::getRootDeviceFile();
 		if (TRUE === $exact) {
-			return (($rootDeviceFile == $deviceFile) ||
+			return ((basename($rootDeviceFile) == basename($deviceFile)) ||
```

Comment:
Why using `basename` here? How does the `$deviceFile` may look like in your tests? I'm asking that to understand what you are trying to approach here.

Reply:
Does `basename` will not counteract the intention of `$exact === TRUE` here?

Reply:
`isRootDeviceFile ` function may not be used anymore at all. 

Agree, that added line is not necessary so does the function itself.